lib/awesome_spawn/command_result_error.rb

```diff
@@ -1,9 +1,13 @@
 module AwesomeSpawn
   class CommandResultError < StandardError
+    include AwesomeSpawn::Logging
```

Comment:
Since you are in the AwesomeSpawn namespace, technically you don't need the `AwesomeSpawn::` part